content/integration/gnome-terminal.md

```diff
@@ -1,12 +1,7 @@
 ---
-title: "Using gnome-terminal with rvm"
+title: "Using gnome-terminal with RVM"
 ---
 
-Other terminals: [LXTerminal](/integration/lxterminal/),
-[XTerm](/integration/xterm/), [Konsole](/integration/konsole/),
-[Zsh + iTerm](/integration/zsh/#zsh-iterm) and
-[anything else](/integration/).
```

Comment:
honestly I'm not sure I can update all the links, try googling for the gnome instruction and see if you recognize all the authors so they can update the link ;)